src/worker/predictive-text-studio-worker-impl.ts

```diff
@@ -82,7 +82,13 @@ export class PredictiveTextStudioWorkerImpl
     name: string,
     contents: File
   ): Promise<number> {
-    const wordlist = await readExcel(await contents.arrayBuffer());
+    let wordlist: WordList = [];
+    if (/\.(tsv)$/i.test(name)) {
+      wordlist = await readTSV(contents);
+    } else {
+      wordlist = await readExcel(await contents.arrayBuffer());
```

Comment:
I wrote some business logic about what kind of rows should be ignored (header rows, commented out rows, etc) for excel files. Is there a way to get both files to share the same kind of business rules functions (i.e. https://github.com/eddieantonio/predictive-text-studio/blob/c0db7a23e0aedb27300caf2821c18d93df5b5ca8/src/worker/read-wordlist.ts#L45)? We could potentially break this out into a separate issue if you'd like.